projects/storefrontlib/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -77,6 +87,25 @@ export class SearchBoxComponent implements OnInit, OnDestroy {
 
   protected subscriptions = new Subscription();
 
+  get isMobile(): Observable<boolean> | undefined {
+    return this.breakpointService?.isDown(BREAKPOINT.sm);
+  }
+
+  get a11ySearchBoxMobileFocusEnabled(): boolean {
```

Comment:
Lets mark this as deprecated along with the cleanup comment since it is public facing and we don't want people using it.

Reply:
Comment added